doc/classes/Basis.xml

```diff
@@ -6,7 +6,12 @@
 	<description>
 		The [Basis] built-in [Variant] type is a 3×3 [url=https://en.wikipedia.org/wiki/Matrix_(mathematics)]matrix[/url] used to represent 3D rotation, scale, and shear. It is frequently used within a [Transform3D].
 		A [Basis] is composed by 3 axis vectors, each representing a column of the matrix: [member x], [member y], and [member z]. The length of each axis ([method Vector3.length]) influences the basis's scale, while the direction of all axes influence the rotation. Usually, these axes are perpendicular to one another. However, when you rotate any axis individually, the basis becomes sheared. Applying a sheared basis to a 3D model will make the model appear distorted.
-		A [Basis] is [b]orthogonal[/b] if its axes are perpendicular to each other. A basis is [b]normalized[/b] if the length of every axis is [code]1[/code]. A basis is [b]uniform[/b] if all axes share the same length (see [method get_scale]). A basis is [b]orthonormal[/b] if it is both orthogonal and normalized, which allows it to only represent rotations. A basis is [b]conformal[/b] if it is both orthogonal and uniform, which ensures it is not distorted.
+		A [Basis] is:
+		- [b]orthogonal[/b] if its axes are perpendicular to each other;
+		- [b]normalized[/b] if the length of every axis is [code]1.0[/code];
+		- [b]uniform[/b] if all axes share the same length (see [method get_scale]);
+		- [b]orthonormal[/b] if it is both orthogonal and normalized, which allows it to only represent rotations (see [method orthonormalized]);
+		- [b]conformal[/b] if it is both orthogonal and uniform, which ensures it is not distorted.
```

Comment:
```suggestion
		- [b]Orthogonal[/b] if its axes are perpendicular to each other;
		- [b]Normalized[/b] if the length of every axis is [code]1.0[/code];
		- [b]Uniform[/b] if all axes are the same length (see [method get_scale]);
		- [b]Orthonormal[/b] if it is both orthogonal and normalized, which allows it to only represent rotations (see [method orthonormalized]);
		- [b]Conformal[/b] if it is both orthogonal and uniform, which ensures it is not distorted.
```
I believe this is how lists like this are formatted elsewhere, and I think "are the same length" reads better

Reply:
I personally tend to be more specific if possible (e.g. _"enable"_ instead of _"use"_) so _"share"_ makes sense to me, but I don't feel too strongly about it.

About lists, we annoyingly don't have a consistent standard. But it feels like it's typical to start in lowercase for each item in the list in this case[^1], but not always[^2][^3].
I'll still do the change and end every line with a dot, instead, just... to be sure. But it would be nice to standardize it someday.

---
[^1]:https://www.purchase.edu/editorial-style-guide/general-style-preferences/punctuation/bulleted-and-numbered-lists/
[^2]:https://english.stackexchange.com/questions/15131/how-to-write-a-bulleted-list
[^3]:https://www.instructionalsolutions.com/blog/bulleted-list-punctuation

Reply:
Yeah, in my experience Godot does tend to use sentence case (capitalized and with a period at the end) for most lists, even in cases where general style guides would not recommend it. Godot really likes sentence case.